lib/symbols.js

```diff
@@ -13,7 +13,13 @@ const keys = {
   kMiddlewares: Symbol('fastify.middlewares'),
   kCanSetNotFoundHandler: Symbol('fastify.canSetNotFoundHandler'),
   kFourOhFourLevelInstance: Symbol('fastify.404LogLevelInstance'),
-  kFourOhFourContext: Symbol('fastify.404ContextKey')
+  kFourOhFourContext: Symbol('fastify.404ContextKey'),
+  kReplySerializer: Symbol('reply.serializer'),
```

Comment:
I think the Symbol name should always be prefixed with `fastify`.
Eg: `fastify.reply.serializer`

@mcollina what do you think?